src/function.rs

```diff
@@ -28,7 +28,7 @@ pub struct FunctionProperties {
     pub named_arguments: Vec<WString>,
 
     /// Description of the function.
-    pub description: WString,
+    pub description: LocalizableString,
```

Comment:
The type no longer expresses that this is currently always dynamic,
but that's definitely worth trading in for extra safety